web-server/routes/guide.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+describe('GET /guide route', function() {
+  it('should render an iframe containing the guide', function(done) {
+    webServer()
+      .get('/guide/')
+      .end(function (err, res) {
+        console.log(err);
+        expect(err).to.be.null;
+        expect(res).to.have.status(200);
+        expect(res.text).to.contain('<title>Guide | Curriculum - Learners Guild</title>')
```

Comment:
I'l like to use something like `cheerio` to  actually parse the HTML rather than just matching text. See what @spencerdezartsmith did :)